runtime/interpreter/value_string.go

```diff
@@ -108,6 +106,74 @@ func stringFunctionFromCharacters(invocation Invocation) Value {
 	return NewUnmeteredStringValue(builder.String())
 }
 
+var StringTypeJoinDefaultSeparator = NewUnmeteredStringValue(",")
```

Comment:
It's probably better to have the separator default to an empty string
```suggestion
var StringTypeJoinDefaultSeparator = EmptyString
```

Reply:
Not applicable.